ui/plugins/eu.esdihumboldt.hale.ui/plugin.xml

```diff
@@ -914,6 +914,24 @@
             name="URL"
             priority="-5">
       </source>
+      <source
+            class="eu.esdihumboldt.hale.ui.io.source.MultipleSchemaFilesSource"
+            description="Load a resource from a file"
+            icon="icons/file_obj.gif"
+            id="eu.esdihumboldt.hale.ui.io.source.multiplefiles"
+            name="MultipleFiles"
```

Comment:
Is that where "From MultipleFiles" comes from in the UI? Maybe "multiple files" would be better then?

Reply:
Yes that was the cause "From MultipleFiles", now I have updated the name.